kafka-ui-react-app/src/components/Connect/New/__tests__/New.spec.tsx

```diff
@@ -93,7 +93,7 @@ describe('New', () => {
       );
     });
 
-    it('redirects to connector details view on successful submit', async () => {
+    xit('redirects to connector details view on successful submit', async () => {
```

Comment:
I could make these tests work by replacing `name` field update in `simulateFormSubmit` with
```typescript
wrapper.find('input[name="name"]').simulate('change', {
  target: { name: 'name', value: 'my-connector' },
});
```